### ISSUE TITLE
Doesn't work with latest NX

### ISSUE THREAD

**Issue by fvanwijk** [samatechtw/nx-vue3-vite#41]
When I create a new empty NX workspace (13.2.3) and then run `npm install nx-vue3-vite --save-dev` I have this error

> npm ERR! code ERESOLVE
> npm ERR! ERESOLVE could not resolve
> npm ERR! 
> npm ERR! While resolving: @nrwl/cypress@13.2.3
> npm ERR! Found: cypress@9.1.0
> npm ERR! node_modules/cypress
> npm ERR!   peer cypress@"^9.1.0" from nx-vue3-vite@0.12.2
> npm ERR!   node_modules/nx-vue3-vite
> npm ERR!     dev nx-vue3-vite@"*" from the root project
> npm ERR! 
> npm ERR! Could not resolve dependency:
> npm ERR! peerOptional cypress@">= 3 < 9" from @nrwl/cypress@13.2.3
> npm ERR! node_modules/@nrwl/cypress
> npm ERR!   dev @nrwl/cypress@"13.2.3" from the root project
> npm ERR! 
> npm ERR! Conflicting peer dependency: cypress@8.7.0
> npm ERR! node_modules/cypress
> npm ERR!   peerOptional cypress@">= 3 < 9" from @nrwl/cypress@13.2.3
> npm ERR!   node_modules/@nrwl/cypress
> npm ERR!     dev @nrwl/cypress@"13.2.3" from the root project
> npm ERR! 
> npm ERR! Fix the upstream dependency conflict, or retry
> npm ERR! this command with --force, or --legacy-peer-deps
> npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

Running again with `--legacy-peer-deps` works.
Nevertheless I'm afraid that if this repo is not being maintained actively, I cannot rely on nx-vue3-vite in the future. It will holding me back from bumping deps to latest.

Then if I run `nx g nx-vue3-vite:app my-app`, I get error
> Cannot find module '@angular-devkit/schematics'

Installing `@nx/angular` and running above command again:

> Cannot convert undefined or null to object

I pretty stuck right now.

**Comment by samatechtw** [comment]
Sorry about that! #3 needs to get done so this doesn't happen in the future.

Currently we're only using the executors in our internal projects, which is why this wasn't caught earlier. I'll try to reproduce/fix this issue by tomorrow.

Looks like the `"@nrwl/cypress": "*"` dep is causing problems, for one. That was inherited from the Nx React example I based this on.

Contributions always welcome!

**Comment by fvanwijk** [comment]
Thanks for the quick reply. Looking forward to the fix. I will try again once it is done.

**Comment by piscis** [comment]
@fvanwijk for a workaround: I added `yarn add -D  @angular-devkit/schematics` to the dev dependency of my workspace and then recreated my app with `nx g nx-vue3-vite:app myapp`

**Comment by samatechtw** [comment]
Sorry for the delay here, PR #42 should resolve this. A couple more generators needed to be tested more extensively before I'm comfortable merging, I'll try to get to it today.

**Comment by samatechtw** [comment]
This should be resolved in `v0.13.0`, let me know if you still experience any issues! The following works for me:

```bash
mkdir test
cd test
npx create-nx-workspace --preset=npm .
npm i -D ../nx-vue3-vite/dist/packages/vue3-vite
npx nx g nx-vue3-vite:app my-app
npx nx build my-app
```

Before closing I will upgrade our internal projects and monitor for any problems.

**Comment by piscis** [comment]
@samatechtw Thx it works

**Comment by samatechtw** [comment]
Fixed in #42, please reopen or create a new issue if there are still problems.
